aospy/test/test_utils_times.py

```diff
@@ -160,87 +161,80 @@ def test_numpy_datetime_range_workaround():
         datetime.datetime(pd.Timestamp.min.year + 4, 1, 1))
 
 
-def test_numpy_datetime_workaround_encode_cf():
-    def create_test_data(days, ref_units, expected_units):
-        # 1095 days corresponds to three years in a noleap calendar
-        # This allows us to generate ranges which straddle the
-        # Timestamp-valid range
-        three_yrs = 1095.
-        time = xr.DataArray([days, days + three_yrs], dims=[TIME_STR])
-        ds = xr.Dataset(coords={TIME_STR: time})
-        ds[TIME_STR].attrs['units'] = ref_units
-        ds[TIME_STR].attrs['calendar'] = 'noleap'
-        actual, min_yr, max_yr = numpy_datetime_workaround_encode_cf(ds)
-
-        time_desired = xr.DataArray([days, days + three_yrs],
-                                    dims=[TIME_STR])
-        desired = xr.Dataset(coords={TIME_STR: time_desired})
-        desired[TIME_STR].attrs['units'] = expected_units
-        desired[TIME_STR].attrs['calendar'] = 'noleap'
-        return actual, min_yr, max_yr, desired
+def _create_test_data(days, ref_units, expected_units):
```

Comment:
Thanks for cleaning up these tests!  Now that this function is no longer a sub-function of `test_numpy_datetime_workaround_encode_cf`, maybe we should name it a little more specifically?  Perhaps something like: `_create_datetime_workaround_test_data`?